com.chartboost.mediation/Runtime/Partners.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using Chartboost.Consent;
+
+namespace Chartboost
+{
+    /// <summary>
+    /// List of Adapter identifiers to be utilized when calling <see cref="ChartboostMediation.StartWithOptions"/> and <see cref="IPartnerConsent"/>
+    /// </summary>
+    public static class Partners
+    {
+        public const string AdColony = "adcolony";
+        public const string AdMob = "admob";
+        public const string AmazonPublisherServices = "amazon_aps";
+        public const string AppLovin = "applovin";
+        public const string MetaAudienceNetwork = "facebook";
+        public const string DigitalTurbineExchange = "fyber";
+        public const string GoogleBidding = "google_googlebidding";
+        public const string InMobi = "inmobi";
+        public const string IronSource = "ironsource";
+        public const string Mintegral = "mintegral";
+        public const string Pangle = "pangle";
+        public const string Tapjoy = "tapjoy";
+        public const string Unity = "unity";
+        public const string Vungle = "vungle";
+        public const string Yahoo = "yahoo";
+        public const string MobileFuse = "mobilefuse";
+        public const string Verve = "verve";
+        public const string HyprMX = "hyprmx";
```

Comment:
Might be a good idea to alphabetize them.

Reply:
Done